docs/http.md

```diff
@@ -535,6 +543,20 @@ step.
   | else                   | Indicates the number of intersections passed until the turn. Example instruction: `at the fourth intersection, turn left` |
   
 
+- `indication`
+  
+  | `type`                 | Description                                                                                                               |
```

Comment:
I think this table is broken. Member should be named `indication` to be consistent with your other namings.